src/test/GenreSelectComponent.test.js

```diff
@@ -0,0 +1,38 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import GenreSelectComponent from '../components/GenreSelectComponent.js';
+
+test('renders values from props', () => {
+    const values = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
+    const current = values[0];
+    const mockOnChange = jest.fn();
+
+    render(<GenreSelectComponent genresList={values} currentGenreInput = {current} onSelect={mockOnChange}></GenreSelectComponent>);
+    values.map(genre => {
```

Comment:
You can add one more test to verify that the count of `li` elements with genres equals the value's length.

Reply:
Thank you, I'll add it in the next PR